packages/profile-sync-controller/src/sdk/user-storage.test.ts

```diff
@@ -23,6 +21,7 @@ import encryption, { createSHA256Hash } from '../shared/encryption';
 import { SHARED_SALT } from '../shared/encryption/constants';
 import { Env } from '../shared/env';
 import { USER_STORAGE_FEATURE_NAMES } from '../shared/storage-schema';
+import type { UserStorageGenericFeatureKey } from 'src/shared/storage-schema';
```

Comment:
Shouldn't this be `../shared/storage-schema`? I guess `src/...` probably works too